### DISCUSSION TITLE
DT/HT rates ranked status discussion

### DISCUSSION THREAD

**Discussion by Givikap120** [ppy/osu#29636] [Polls]
As you can know - RA (Rate Adjust, or custom rates for DT/HT mods) is getting ranked soon. Considering this I want to discussed what rates exactly are going to be ranked.

Current state of the custom rates:
- Only 0.1x increments result in score bonus/penalty for both DT and HT
- You can adjust rate by increments of 0.05x using CTRL + Arrow Up/Down

So there are 3 options of ranked rates:
1) Rank increments of 0.05x for HT (to have 0.75x ranked), and 0.1x increments for DT
2) Rank all increments of 0.05x for both DT / HT
3) Rank all rates for both DT / HT

Arguments in favor of first option over second option:
- minimal amount of min/maxing
- the most convenient leaderboard filtering
- consistency with score multiplier (only for DT now, but I believe HT will have per 0.05x score multiplier in future in anyway)

Arguments in favor of second option over first option:
- more smooth difficulty increase for stream maps, especially high BPM stream maps
- same rate increments for both DT and HT
- consistency with arrow keybind

Third option only advantage is having maximal freedom, but it have certain disadvantages:
- maximal amount of min/maxing
- major leaderboard filtering complication
- pp system is unstable for small rate changes (https://github.com/ppy/osu/issues/25254)

I personally believe that 2nd option is the best compromise between freedom and issues that come with it.

**Comment by cl8n** [comment]
I voted "0.05x for both" but I think this is going to be weird or at least esoteric that the steps for score multipliers don't match the steps for pp awarding. it would be nice if those were the same

**Reply by Givikap120** [reply]
if 0.05x for both is picked - then when multiplier change will arrive: it will be "per 0.05x" for both
even in theoretic case of first variant winning it still would be changed because HT currently also have per 0.1x step multiplier, what doesn't makes any sense
